### ISSUE TITLE
Infinite-Loop with Midi-QOL

### ISSUE THREAD

**Issue by Mist-Hunter** [dev7355608/perfect-vision#54]
I am suspicious that this problem lies with Midi-QOL, but I'm cross posting here in case any others haven't identified the relationship. Issue posted on GitLab @ Midi-QOL https://gitlab.com/tposney/midi-qol/-/issues/216#note_492444667

With Midi-QOL + Perfect-vision enabled, the browser enters a reload loop that is very difficult to stop. I have to repeated try and access the modules > disable modules > save option prior to the reload loop.

**Comment by dev7355608** [comment]
I thought the infinite reload loop was fixed with v1.7.1/v1.7.2. Is your Perfect Vision up to date?

**Comment by Mist-Hunter** [comment]
This is the first time I've encountered it, and it's definitely perfect-vision--v1.7.2. I noticed 5e just updated to 1.24 in the last day or two. I have no idea which module it is, though I am seeing some weirdness in midi-qol that I posted in that bug report.

**Comment by dev7355608** [comment]
Did you try to delete the Perfect Vision module and completely reinstall it? Maybe the update failed somehow. I not sure how the reload loop could still happen with v1.7.2.

**Comment by Mist-Hunter** [comment]
At your suggestion, I just tried it. I experienced the same issue. Earlier today I did the same (delete re-install) with midi-qol. Oddly, I just discovered this is not affecting my firefox. I'm not sure what's going on, and it may not be perfect-vision, but I do see someone else having the same issue so I'm doubt it's just a chache issue or something. Thanks for the help, and the module either way!
